### DISCUSSION TITLE
VIM browsing

### DISCUSSION THREAD

**Discussion by xplshn** [minbrowser/min#2298] [Ideas]
Add VIM-like shortcuts for keyboard only browsing - Just like https://github.com/jun7/wyeb has, or add chromium extension's support and add https://github.com/philc/vimium/wiki by default, so users can be prompted to enable vim browsing.

There's also https://github.com/Jelmerro/Vieb which is based upon Electron.

**Comment by sndrkrly** [comment]
https://github.com/PalmerAL/min-vim-mode
